.kitchen.yml

```diff
@@ -22,10 +14,31 @@ provisioner:
         - elasticsearch
 
 platforms:
+  # update base ubuntu image and install
+  # software-properties-common  for add-apt-repository
+  <% @ubuntu_provision_commands = [
+    'locale-gen en_US.UTF-8', 'apt-get update',
+    'apt-get install -y software-properties-common',
+    'add-apt-repository -y ppa:openjdk-r/ppa', 'apt-get update',
+    'apt-get install -y openjdk-8-jre-headless', 'update-ca-certificates -f'] %>
   - name: ubuntu-14.04
+    driver_config:
+      provision_command:
+        <% @ubuntu_provision_commands.each do |provision_command| %>
+        - <%= provision_command %>
+        <% end %>
   - name: ubuntu-16.04
     driver_config:
       run_command: /sbin/init
+      provision_command:
+        <% @ubuntu_provision_commands.each do |provision_command| %>
+        - <%= provision_command %>
+        <% end %>
+  - name: centos-6
+    driver_config:
+      box: rchrd/centos-6-x64-salt
```

Comment:
No worries! You used the right name. So, an appropriate image was used. It will be fine if you get rid of the `driver_config` here.